packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Updates minimum Flutter version to 2.10.
+* Added setMaxVideoResolution method to support different video resolutions while picking a video stream from a source with different tracks
```

Comment:
Nit:
```suggestion
* Added setMaxVideoResolution method to support different video resolutions while picking a video stream from a source with different tracks.
```

Also, I believe this change deserves a version bump. That would go for the other platforms, as well.